phys/module_mixactivate.F

```diff
@@ -1163,36 +1165,65 @@ subroutine mixactivate(  msectional,            &
               chem(i,kts:kte,j,lnumcw)= raercol(kts:kte,lnumcw,nnew)*scale
               chem(i,kts:kte,j,lnum)= raercol(kts:kte,lnum,nnew)*scale
            endif
+           tmp_vol_mr(kts:kte) = 0.0
            do l=1,ncomp(n)
               lmass=massptr_aer(l,m,n,ai_phase)
               lmasscw=massptr_aer(l,m,n,cw_phase)
-!          scale = mwdry/mw_aer(l,n)
+              !          scale = mwdry/mw_aer(l,n)
```

Comment:
Just remove this commented out line completely.

Reply:
Should be taken care of.